pad.c

```diff
@@ -2844,6 +2844,91 @@ Perl_padname_dup(pTHX_ PADNAME *src, CLONE_PARAMS *param)
 
 #endif /* USE_ITHREADS */
 
+/*
+=for apidoc_section $lexer
+=for apidoc suspend_compcv
+
+Implements part of the concept of a "suspended complication CV"; which can be
```

Comment:
Will the concept of a "suspended complication CV" be explained in user-facing documentation in a subsequent commit (*i.e.,* `pod/perlsomething.pod`)?

Also, what is the point of the semicolon in the line above.  Shouldn't it either be a comma or rewritten as `"suspended complication CV." This can be`

Reply:
> Will the concept of a "suspended complication CV" be explained in user-facing documentation in a subsequent commit (_i.e.,_ `pod/perlsomething.pod`)?

Not sure what you mean "user-facing"... it is explained in the rest of the sentence after the semicolon (to be a comma, as per below)

> Also, what is the point of the semicolon in the line above. Shouldn't it either be a comma or rewritten as `"suspended complication CV." This can be`

Ahyes a comma probably reads better.